src/main/java/seedu/address/model/property/PropertyAddress.java

```diff
@@ -5,9 +5,9 @@
 
 /**
  * Represents a Property's address in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ * Guarantees: immutable; is valid as declared in {@link #isValidPropertyAddress(String)}
  */
-public class Address {
+public class PropertyAddress {
```

Comment:
We can probably move the Address class out into the model folder, so both Person and Property can share the same address validation class

Reply:
resolved in latest commit [3c58a46](https://github.com/AY2223S1-CS2103T-F12-1/tp/pull/70/commits/3c58a46dac9abb082c1ce3bf0c16f5c59004866a)